openapi-bal-task-plugin/src/main/java/io/ballerina/openapi/bal/tool/Constants.java

```diff
@@ -37,6 +37,7 @@ public class Constants {
     public static final String STATUS_CODE_BINDING = "statusCodeBinding";
     public static final String MOCK = "mock";
     public static final String SINGLE_FILE = "singleFile";
+    public static final String IS_SANITIZED_OAS = "isUsingSanitizedOas";
```

Comment:
Shall we just say `useSanitizedOas` or `useSanitizedSchema` as the option key?

Reply:
This is just a mock flag and here we wish to modify all the OpenAPI Specifications therefore use `useSanitizedOas`.  We will be reviewing the name of the flag.